### PR TITLE
Update iOS SDK v2.1.8 release notes

### PR DESCRIPTION
**Pull request by vonageCI**
iOS SDK v2.1.8 notes added to `release-notes.md`.

### CODE REVIEW

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ navigation_weight: 0
 # Release Notes
 
 
+## 2.1.8 - 2020-04-02
+
+### Added
+
+- Add `useFirstIceCandidate` parameters to `NXMClientConfig`
+
+```
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:16]: Add language specified e.g. `objective-c` to get syntax colouring.

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ navigation_weight: 0
...
+NXMClientConfig *config = [[NXMClientConfig alloc] initWithApiUrl:restUrl
+                                                     websocketUrl:wsUrl
+                                                           ipsUrl:ipsUrl
+                                             useFirstIceCandidate:NO];
+```
+
+
 ## 2.1.5 - 2020-03-18
 
 ### Internal
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:26]: Always have new line after heading.

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md (hunk 1 above)
Comment (pardel) [reply _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:16]: @ash30 Could you please delegate accordingly?

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md (hunk 2 above)
Comment (pardel) [reply _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:26]: @ash30 Could you please delegate the tasks accordingly?

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,20 @@ navigation_weight: 0
...
+
+- Add `useFirstIceCandidate` parameters to `NXMClientConfig`
+
+```
+NXMClientConfig *config = [[NXMClientConfig alloc] initWithApiUrl:restUrl
+                                                     websocketUrl:wsUrl
+                                                           ipsUrl:ipsUrl
+                                             useFirstIceCandidate:NO];
+```
+
```
Comment (horothesun) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:22]:
```suggestion
```

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md (hunk 1 above)
Comment (horothesun) [reply _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md:16]:
```suggestion
```objective_c
```

**Comment by tbedford** [comment]
@fabianrbz is investigating a Ruby build issue.

**Pull request merged**